bytes/bytes.mbt

```diff
@@ -29,3 +29,11 @@ pub fn Bytes::from_array(arr : Array[Int]) -> Bytes {
   }
   rv
 }
+
+pub fn to_array(self : Bytes) -> Array[Int] {
+  let rv = Array::with_capacity(self.length())
```

Comment:
1. it is more efficient to 
`Array::make(self.length(),0)` and `rv[i]=.. ` directly
2. Array::with_capacity API is a bit weird, can it be unified with 
 Array::init(~capcaity:Int)